tt_metal/impl/kernels/kernel.cpp

```diff
@@ -213,6 +213,7 @@ RuntimeArgsData &Kernel::common_runtime_args_data() { return this->common_runtim
 void Kernel::validate_runtime_args_size(
     size_t num_unique_rt_args, size_t num_common_rt_args, const CoreCoord &logical_core) {
     uint32_t total_rt_args = (num_unique_rt_args + num_common_rt_args);
+    uint32_t idle_eth_max_runtime_args = hal.get_dev_size(HalProgrammableCoreType::ACTIVE_ETH, HalL1MemAddrType::KERNEL_CONFIG) / sizeof(uint32_t);
```

Comment:
@abhullar-tt could you please double check this for me?

My change here is a brain dead substitution, however its bizarre to me that the variable was named "idle_eth_*" and I have to replace with a parameter defined as part of "ACTIVE_ETH" core.

Reply:
Hmmm... current change causes segmentation fault in all grayskull test cases.

Reply:
I think I fixed ... grayskull doesn't have a hal entry for active ethernet core.